<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ip地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIp"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonStart">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textShowEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QTextBrowser" name="textShowClientConnect"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditInput"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
